channeldb/channel.go

```diff
@@ -596,34 +596,23 @@ func (c *OpenChannel) RefreshShortChanID() error {
 	c.Lock()
 	defer c.Unlock()
 
-	var sid lnwire.ShortChannelID
-	var isPending bool
 	err := c.Db.View(func(tx *bbolt.Tx) error {
 		chanBucket, err := fetchChanBucket(
 			tx, c.IdentityPub, &c.FundingOutpoint, c.ChainHash,
 		)
 		if err != nil {
 			return err
 		}
-
-		channel, err := fetchOpenChannel(chanBucket, &c.FundingOutpoint)
-		if err != nil {
-			return err
-		}
-
-		sid = channel.ShortChannelID
-		isPending = channel.IsPending
+		if err := fetchChanInfo(chanBucket, c); err != nil {
```

Comment:
this should have a comment, and an updated godoc on the method. Also looks like it can be squashed with the previous commit.

Reply:
fixed.